GETTING_STARTED.md

```diff
@@ -1,125 +1,120 @@
+```python
+query = "FOR doc IN employees SORT doc.age LIMIT 2, 4 RETURN doc"
+cursor = client.execute_query(query=query)
+docs = [document for document in cursor]
+```
 
-# --------------------------------------------------------------
-print("Delete Collection...")
-# Delete Collection
-client.delete_collection(name=collname)
+[Grouping](https://macrometa.com/docs/queryworkers/c8ql/examples/grouping) results is a common operation when retrieving data. To group results by arbitrary criteria, C8QL provides the [COLLECT](https://macrometa.com/docs/queryworkers/c8ql/operations/collect) keyword. [COLLECT](https://macrometa.com/docs/queryworkers/c8ql/operations/collect) will perform a grouping, but no aggregation. In the following example let's group by **age** and return only the **first name** of the employee using the  expansion operator [*].
+
+```python
+query = "FOR doc IN employees COLLECT age = doc.age INTO employeesByAge RETURN {age, employee: employeesByAge[*].doc.firstname}"
+cursor = client.execute_query(query=query)
+docs = [document for document in cursor]
 ```
 
+
 ## Returning all data records inside the collection via batches
 The Macrometa GDN has a default limit on how many documents can be returned per query. Usually, the default limit is 1,000 documents per query (This default limit is subject to changes).
 The following method should be used to retrieve data from a collection in batches despite this default limit of 1,000 documents per query.
 
-You need to specify the collection name in the `collection_name` parameter
+[See the full example here](./examples/get_data_in_batches.py)
 ```python
-client.get_all_documents(collection_name="employees")
-```
+collection_name = "result"
+document_count = fabric.collection(collection_name).count()
+iterations = int(math.ceil(document_count / 1000))
+data = []
 
-## Returning all data records returned by any query via batches
-The Macrometa GDN has a default limit on how many documents can be returned per query. Usually, the default limit is 1,000 documents per query (This default limit is subject to changes). 
-The following method should be used to retrieve all the data from any query via batches despite this default limit of 1,000 documents per query.
+for i in range(iterations):
+    a = i * 1000
+    query = "FOR doc IN {} LIMIT {}, {} RETURN doc".format(collection_name, a, 1000)
+    cursor = fabric.c8ql.execute(query, count=True, batch_size=1000)
+    data.append(cursor.batch())
 
-You need to specify your query in the `query` parameter
-```python
-client.get_all_batches(query="FOR doc IN employees FILTER doc.email LIKE '%macrometa.io' RETURN doc")
+# Clean the data
+flat_data = [item for sublist in data for item in sublist]
 ```
 
 # Query Workers
```

Comment:
Shouldn't we add a complete example code for query workers too .. show casing all the below methods?